File: ass1_q2.py
print('Program to find factorial of integer n')

def factorial(n):
    if n < 0:
        print('Factorial is not define for negative number')
    else:
        f = 1
        for i in range(1,n+1):
            f = f*i
        print(f'Factorial of {n}: {f}')


n = int(input('Enter integer n: '))
factorial(n)

File: ass1_q2.cpp
#include <iostream>
using namespace std;

int main()
{   
    cout << "Program to find factorial of integer n" <<endl;
    cout << "Enter a integer n: ";

    int n, f = 1;
    cin >> n;

    if (n < 0){
        cout << "Please enter the non negative integer number" <<endl;
    }
    else{
        for (int i = 1; i <= n; ++i){
            f = f*i;
        }
        cout << "Factorial of "<< n << " is " << f <<endl;
    }

    return 0;
}